### DISCUSSION TITLE
How to run generator.py in Github.

### DISCUSSION THREAD

**Discussion by marcothijs** [dodona-edu/feedback-and-support#50] [Questions - Vragen]
Dear,
I've seen a generator.py file in some of the Github content of the exercises of Programmerusleerling (chapters/16 text files/08 exercises/01 counting words/preparation/generator.py). It is not clear how to run or use it. Thanks for your help. 

Regards,
Marco

**Comment by bmesuere** [comment]
These are files created by @pdawyndt when converting the book to a dodona-compatible format. In addition, they use the old python judge, so you can safely ignore them.

**Reply by marcothijs** [reply]
Thanks for your reply. I see that this is the case for this example but there are also generator.py files in exercises that use the TESTed judge. And for the turtle exercises it is used to generate the solution picture. My interest is more general, how do I run a .py script within Github.

Regards,
Marco

**Reply by pdawyndt** [reply]
> Thanks for your reply. I see that this is the case for this example but there are also generator.py files in exercises that use the TESTed judge. And for the turtle exercises it is used to generate the solution picture. My interest is more general, how do I run a .py script within Github.
> 
> Regards, Marco

I run these scripts locally on my machine, not on GitHub.
